lib/utils/fetchSDP.tsx

```diff
@@ -0,0 +1,25 @@
+import { WsSdpPipeline, VideoMedia } from 'media-stream-library'
```

Comment:
This files contains no `tsx`, so I suggest renaming it to `ts`.

Reply:
Fixed.